editor/scene_tree_dock.h

```diff
@@ -112,6 +112,8 @@ class SceneTreeDock : public VBoxContainer {
 	ToolButton *button_create_script;
 	ToolButton *button_clear_script;
 
+	Button* button_3d;
```

Comment:
Also, this `*` should be with `button_3d`, not `Button`.

Reply:
I am sorry for the inconveniences caused. I am new to open source development have usually worked on small scale projects.

Reply:
Don't worry, we all have been there. You'll get used to it in no time!